### PR TITLE
add query-param for websocket-readme

### PR DESCRIPTION
**Pull request by rdhabalia**
### Motivation

Add query-param info in websocket readme.

### CODE REVIEW

File: site/docs/latest/clients/WebSocket.md [hunk 1]
```diff
@@ -81,6 +81,19 @@ The producer endpoint requires you to specify a {% popover property %}, {% popov
 
 {% endpoint ws://broker-service-url:8080/ws/producer/persistent/:property/:cluster/:namespace/:topic %}
 
+##### Query param
+
+Key | Type | Required? | Explanation
+:---|:-----|:----------|:-----------
+`sendTimeoutMillis` | string | no | Send timeout (default: 30 secs)
+`batchingEnabled` | string | no | Enable batching of messages(default:disable)
+`batchingMaxMessages` | string | no | Maximum number of messages permitted in a batch
```
Comment (merlimat) [inline site/docs/latest/clients/WebSocket.md]: type: int, default 1000

File: site/docs/latest/clients/WebSocket.md [hunk 2]
```diff
@@ -81,6 +81,19 @@ The producer endpoint requires you to specify a {% popover property %}, {% popov
 
 {% endpoint ws://broker-service-url:8080/ws/producer/persistent/:property/:cluster/:namespace/:topic %}
 
+##### Query param
+
+Key | Type | Required? | Explanation
+:---|:-----|:----------|:-----------
+`sendTimeoutMillis` | string | no | Send timeout (default: 30 secs)
+`batchingEnabled` | string | no | Enable batching of messages(default:disable)
```
Comment (merlimat) [inline site/docs/latest/clients/WebSocket.md]: Shouldn't the type be `boolean` here and the the default `false`?

File: site/docs/latest/clients/WebSocket.md [hunk 3]
```diff
@@ -81,6 +81,19 @@ The producer endpoint requires you to specify a {% popover property %}, {% popov
 
 {% endpoint ws://broker-service-url:8080/ws/producer/persistent/:property/:cluster/:namespace/:topic %}
 
+##### Query param
+
+Key | Type | Required? | Explanation
+:---|:-----|:----------|:-----------
+`sendTimeoutMillis` | string | no | Send timeout (default: 30 secs)
```
Comment (merlimat) [inline site/docs/latest/clients/WebSocket.md]: type `int`

File: site/docs/latest/clients/WebSocket.md [hunk 4]
```diff
@@ -81,6 +81,19 @@ The producer endpoint requires you to specify a {% popover property %}, {% popov
...
+`batchingEnabled` | string | no | Enable batching of messages(default:disable)
+`batchingMaxMessages` | string | no | Maximum number of messages permitted in a batch
+`maxPendingMessages` | string | no | Set the max size of the internal-queue holding the messages
+`batchingMaxPublishDelay` | string | no | Time period within which the messages will be batched
+`messageRoutingMode` | string | no | Message [routing mode](https://pulsar.incubator.apache.org/api/client/index.html?org/apache/pulsar/client/api/ProducerConfiguration.MessageRoutingMode.html) for the partitioned producer
+`compressionType` | string | no | Compression [type](https://pulsar.incubator.apache.org/api/client/index.html?org/apache/pulsar/client/api/CompressionType.html)
```
Comment (merlimat) [inline site/docs/latest/clients/WebSocket.md]: Can we also list the available options here (`ZLIB`, `LZ4`)

File: site/docs/latest/clients/WebSocket.md (hunk 3 above)
Comment (rdhabalia) [reply site/docs/latest/clients/WebSocket.md]: oops.. I missed to change data-type everywhere.. :-(

File: site/docs/latest/clients/WebSocket.md [hunk 5]
```diff
@@ -132,6 +145,15 @@ The consumer endpoint requires you to specify a {% popover property %}, {% popov
 
 {% endpoint ws://broker-service-url:8080/ws/consumer/persistent/:property/:cluster/:namespace/:topic/:subscription %}
 
+##### Query param
+
+Key | Type | Required? | Explanation
+:---|:-----|:----------|:-----------
+`ackTimeoutMillis` | long | no | Set the timeout for unacked messages (default: 1000)
```
Comment (massakam) [inline site/docs/latest/clients/WebSocket.md]: It seems that the default value of `ackTimeoutMillis` is 0.
https://github.com/apache/incubator-pulsar/blob/bace0b47ada0d107f33d26ac282e1d6fb0a43156/pulsar-websocket/src/main/java/org/apache/pulsar/websocket/ConsumerHandler.java#L244
https://github.com/apache/incubator-pulsar/blob/bace0b47ada0d107f33d26ac282e1d6fb0a43156/pulsar-client/src/main/java/org/apache/pulsar/client/api/ConsumerConfiguration.java#L51

Comment (rdhabalia) [reply site/docs/latest/clients/WebSocket.md]: yes, by mistake I copied `long minAckTimeoutMillis = 1000` time. will fix it.

**Comment by merlimat** [comment]
retest this please

**Pull request merged**